edx_rest_framework_extensions/auth/jwt/authentication.py

```diff
@@ -67,6 +70,13 @@ def authenticate(self, request):
             if not user_and_auth:
                 return user_and_auth
 
+            # Fail authentication if user disabled
+            user = user_and_auth[0]
+            if user and isinstance(user, get_user_model()):
+                if not user.has_usable_password():
+                    msg = 'User disabled by admin: %s' % user.get_username()
+                    raise exceptions.AuthenticationFailed(code=JWT_USER_DISABLED_ERROR, detail=msg)
```

Comment:
Will we do the same change in error and log info we did elsewhere?

Reply:
Done